@vueuse/core

### Describe the bug

I use @vueuse/core in my project. Release 0.26.16 is incompatible due to a version conflict:

```
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: bootstrap-vue-next@0.26.16
npm error Found: @vueuse/core@12.0.0
npm error node_modules/@vueuse/core
npm error   @vueuse/core@"^12.0.0" from the root project
npm error   @vueuse/core@"12.0.0" from @vueuse/components@12.0.0
npm error   node_modules/@vueuse/components
npm error     @vueuse/components@"^12.0.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peerOptional @vueuse/core@"^11.2.0" from bootstrap-vue-next@0.26.16
npm error node_modules/bootstrap-vue-next
npm error   bootstrap-vue-next@"^0.26.16" from the root project
npm error
npm error Conflicting peer dependency: @vueuse/core@11.3.0
npm error node_modules/@vueuse/core
npm error   peerOptional @vueuse/core@"^11.2.0" from bootstrap-vue-next@0.26.16
npm error   node_modules/bootstrap-vue-next
npm error     bootstrap-vue-next@"^0.26.16" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution
```

### Reproduction

Require @vueuse/core

### Used Package Manager

npm

--- a/apps/docs/package.json
+++ b/apps/docs/package.json
@@ -23,7 +23,7 @@
     "@vue/eslint-config-prettier": "^10.1.0",
     "@vue/eslint-config-typescript": "^14.1.3",
     "@vue/tsconfig": "^0.6.0",
-    "@vueuse/core": "^11.2.0",
+    "@vueuse/core": "^12.0.0",
     "bootstrap": "^5.3.3",
     "bootstrap-vue-next": "workspace:^",
     "cross-env": "^7.0.3",

--- a/apps/playground/package.json
+++ b/apps/playground/package.json
@@ -11,7 +11,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@vueuse/core": "^11.2.0",
+    "@vueuse/core": "^12.0.0",
     "bootstrap-vue-next": "workspace:^",
     "vue": "^3.5.13",
     "vue-router": "^4.4.5"

--- a/packages/bootstrap-vue-next/package.json
+++ b/packages/bootstrap-vue-next/package.json
@@ -144,7 +144,7 @@
   },
   "peerDependencies": {
     "@floating-ui/vue": "^1.1.5",
-    "@vueuse/core": "^11.2.0",
+    "@vueuse/core": "^12.0.0",
     "vue": "^3.5.13",
     "vue-router": "^4.4.5"
   },
@@ -173,7 +173,7 @@
     "@vue/runtime-core": "^3.5.13",
     "@vue/test-utils": "^2.4.6",
     "@vue/tsconfig": "^0.6.0",
-    "@vueuse/core": "^11.2.0",
+    "@vueuse/core": "^12.0.0",
     "@vueuse/integrations": "^11.2.0",
     "@vueuse/shared": "^11.2.0",
     "bootstrap": "^5.3.3",

